libraries/AP_LandingGear/AP_LandingGear.cpp

```diff
@@ -121,6 +140,9 @@ void AP_LandingGear::deploy()
 
     // set deployed flag
     _deployed = true;
+    _have_changed = true;
```

Comment:
is it my imagination or is _have_changed never set to false anywhere?

Reply:
that's correct, it starts as false. We need to know if the servo has ever been set